.github/workflows/dockerhub_deployment.yml

```diff
@@ -0,0 +1,22 @@
+# SPDX-FileCopyrightText: 2021 Alliander N.V.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+name: Docker Hub Deployment
+
+on: push
```

Comment:
Should there be a filter on "develop" and "main"?
At least not on all the feature branches.

And maybe just go for the "release" event as described in your comment. Mention the solution during the refinement and we will see in the future if this is the way to go.

Reply:
I removed the on push, and replaced it with the following:

```yaml
on:
  release:
    types: [released]
```

According to https://docs.github.com/en/actions/reference/events-that-trigger-workflows, it should now be triggered on when a release is 'released'.